plugins/home/api-report.md

```diff
@@ -66,8 +70,17 @@ export function createCardExtension<T>(options: {
   title: string;
   components: () => Promise<ComponentParts>;
   name?: string;
+  data?: Record<string, any>;
```

Comment:
Component data is not indented to be a user-facing (user being a plugin developer) feature. Let's wrap this up in something like an explicit component data key for storing settings.

Reply:
Yeah, will do!